Sources/SwiftFrameCore/Workers/GraphicsContext.swift

```diff
@@ -0,0 +1,37 @@
+import AppKit
+import CoreGraphics
+import Foundation
+
+class GraphicsContext {
+
+    let cg: CGContext
```

Comment:
I'm happy to rename this and `ci` below to something more verbose. Wasn't sure if the usage of this class is limited enough to always be able to know what `cg` and `ci` are

Reply:
Technically ok, so can leave as is. Personally, I like `cgContext` a little more because it's still somewhat short but less ambiguous.

Reply:
Seeing the `context.cg.saveGState()` below now. I guess that's an argument for keeping the ultra-short name.